R/MsExperiment-functions.R

```diff
@@ -175,7 +175,15 @@
 .link_matrix <- function(from = integer(), to = integer()) {
     res <- lapply(from, function(x) which(to == x))
     ls <- lengths(res)
-    cbind(rep(seq_along(from), ls), unlist(res[ls > 0], use.names = FALSE))
+    cbind(rep(seq_along(from), ls), unlist(res, use.names = FALSE))
+}
+
+#' @importFrom S4Vectors findMatches
+#'
+#' @noRd
+.link_matrix2 <- function(from = integer(), to = integer()) {
```

Comment:
Should become `link_matrix` and the old `link_matrix` should be removed.

Reply:
Done